parm/aqm.rc

```diff
@@ -29,11 +29,9 @@ ctm_pmdiag: true
 #
 #  Input emissions
 #
-{%- if do_aqm_canopy %}
-emission_sources: anthro fire canopy
-{%- else %}
 emission_sources: anthro fire
-{%- endif %}
+{%- if do_aqm_canopy %} canopy{% endif %}
+{%- if do_aqm_fengsha %} fengsha{% endif %}
```

Comment:
This method is simpler and makes it easier to add others. @bbakernoaa 

<details><summary>Demo:</summary>

```python
import jinja2 as j2


s = """\
#
#  Input emissions
#
emission_sources: anthro fire
{%- if do_aqm_canopy %} canopy{% endif %}
{%- if do_aqm_fengsha %} fengsha{% endif %}

...

"""

template = j2.Environment().from_string(s)

print(template.render({"do_aqm_canopy": True, "do_aqm_fengsha": True}))
print(template.render({"do_aqm_canopy": True, "do_aqm_fengsha": False}))
print(template.render({"do_aqm_canopy": False, "do_aqm_fengsha": True}))
print(template.render({"do_aqm_canopy": False, "do_aqm_fengsha": False}))
```
```
#
#  Input emissions
#
emission_sources: anthro fire canopy fengsha

...

#
#  Input emissions
#
emission_sources: anthro fire canopy

...

#
#  Input emissions
#
emission_sources: anthro fire fengsha

...

#
#  Input emissions
#
emission_sources: anthro fire

...

```
</summary>

Reply:
sweet!

Reply:
if this works we may want to push this change to the authoritative

Reply:
Sounds good, I'm going to merge this then and then merge the upstream changes and we can test.